realm/realm-library/src/syncIntegrationTest/java/io/realm/objectserver/QueryBasedSyncTests.java

```diff
@@ -296,6 +300,42 @@ public void clearTable() {
         looperThread.testComplete();
     }
 
+    @Test
+    @RunTestInLooperThread
+    @Ignore("FIXME: Enable when https://github.com/realm/realm-js/issues/1971 is fixed")
```

Comment:
I think you're referencing the wrong issue, the one used concerns custom SSL conf